src/itemsmanagerworker.cpp

```diff
@@ -277,6 +279,7 @@ void ItemsManagerWorker::ParseItems(rapidjson::Value *value_ptr, const ItemLocat
         location.ToItemJson(&item, alloc);
         items_as_string_ += Util::RapidjsonSerialize(item) + ",";
         items_.push_back(std::make_shared<Item>(item));
+        app_.currency_manager().ParseSingleItem(items_.back());
```

Comment:
It's better to update all items' currency values at once when they're refreshed, should be done in `Application::OnItemsRefreshed`.